WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -372,6 +391,20 @@ class PluginViewModel: Observable {
             ])
     }
 
+    private func confirmRegisterDomainAlert() -> UIAlertController {
+        let title = NSLocalizedString("Install Plugin", comment: "Install Plugin dialog title.")
+        let message = NSLocalizedString("To install plugins you need to have a custom domain associated with your site", comment: "Install Plugin dialog text.")
```

Comment:
Thank you for not copying the typo in the design. 😄 

I know this isn't in the design, but would you mind putting a period at the end of this sentence? Thank you!

Reply:
Sure, done.